TestFoundation/TestURL.swift

```diff
@@ -224,10 +231,19 @@ class TestURL : XCTestCase {
     
     static let gBaseTemporaryDirectoryPath = NSTemporaryDirectory()
     static var gBaseCurrentWorkingDirectoryPath : String {
+#if os(Windows)
```

Comment:
Unless this is testing that method, all of this can be replaced with `FileManager.default.currentDirectoryPath`.